templates/views/director_profile.pug

```diff
@@ -3,7 +3,7 @@ include ../mixins/snippets
 include ../mixins/movie-listing
 
 block content
-  #title-container.bg-green.pa3.shadow-5.z-1.light-gray
+  #title-container.pa3.shadow-5.z-1.light-gray(class=`bg-${color}`)
```

Comment:
Finally,

Can we change the name of this variable (locals.color) to something more specific?

Something like primaryColor ..?